### DISCUSSION TITLE
Unable to build with Platform 2024.08.10 for ESP32

### DISCUSSION THREAD

**Discussion by ZioFabry** [arendst/Tasmota#21902] [Support request] [answered]
Anyone have the same issue ?

```
c:/users/ziofa/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/12.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio/build/tasmota32-biopdu/lib3d0/libWire.a: error adding symbols: archive has no index; run ranlib to add one
collect2.exe: error: ld returned 1 exit status
*** [.pio\build\tasmota32-biopdu\firmware.elf] Error 1
```

No problem on Platform 2024.07.11

**Comment by Jason2866** [comment, marked as answer]
Seems your Platformio setup is broken. Close VSC. Delete hidden folder `.platformio` and the hidden cache folder in the Tasmota projects folder. Start VSC and follow the shown commands.

**Comment by ZioFabry** [comment]
after several attempt of clean & uninstall... I succesufully builed with 2024.08.10.... thanks
